### ISSUE TITLE
Strange rendering in matrices written with latexmath

### ISSUE THREAD

**Issue by manuelfuenmayor** [plurimath/plurimath#201]
*In relation to https://github.com/metanorma/iso-10303-detached-docs/pull/233*

### Sample 1

```
[latexmath%unnumbered]
++++
   \left(
\begin{array}{cccccc}
{\displaystyle \frac{1}{E}} & -{\displaystyle \frac{\nu}{E}} & -{\displaystyle \frac{\nu}{E}} & 0 & 0 & 0 \\
     & {\displaystyle \frac{1}{E}}  & -{\displaystyle \frac{\nu}{E}} & 0 & 0 & 0 \\
     & & {\displaystyle \frac{1}{E}} & 0 & 0 & 0 \\
     & & &  {\displaystyle \frac{1}{G}}  & 0 & 0 \\
     & \mbox{symmetric} &  &  & {\displaystyle \frac{1}{G}} & 0 \\
     & & & & & {\displaystyle \frac{1}{G}}
\end{array}
\right)
++++
```

renders:

PDF:
![eq-1-pdf](https://github.com/plurimath/plurimath/assets/33627611/7266f2b8-cc12-462e-8969-87dc5cc52c93)

HTML:
![eq-1-html](https://github.com/plurimath/plurimath/assets/33627611/5e42c530-9493-43c8-99b5-6b7bba8b44e4)

Original:
![eq-1-original](https://github.com/plurimath/plurimath/assets/33627611/53ad9c06-ab85-4e77-9f5e-d24a79fd6a03)

### Sample 2

```
[latexmath%unnumbered]
++++
\begin{array}{cc}
\left(    \begin{array}{ccccccc}
 & & & & & & \\
 & +k & & & & -k  \\
 & & & & & & \\
 & & & & & & \\
 & & & & & & \\
 & -k & & & & +k
    \end{array}  \right)   &
  \begin{array}{cc}
   & \\
   \cdots & \mbox{degree of freedom 1, node 1} \\
   & \\
   & \\
   & \\
   \cdots & \mbox{degree of freedom 2, node 2}
  \end{array}      \\
      & \\
 \begin{array}{cccccc}
    \vdots & & & & & \vdots
     \end{array}  & \\
     \begin{array}{cccc}
& \mbox{degree of}  & \mbox{degree of}  & \\
& \mbox{freedom 1,} & \mbox{freedom 2,} & \\
& \mbox{node 1} & \mbox{node 2}  &
    \end{array}    &
\end{array}
++++

```

renders

PDF:
![eq-2-pdf](https://github.com/plurimath/plurimath/assets/33627611/4dd25281-cc46-4f3f-87b3-1584a2122391)

HTML:
![eq-2-html](https://github.com/plurimath/plurimath/assets/33627611/684691a4-cc3b-4da4-8096-0ce0be6214db)

Original:
![eq-2-original](https://github.com/plurimath/plurimath/assets/33627611/087ff6e9-62ca-4769-a62a-9400ec37c73f)

### To reproduce:

Use this sample file: 
[equations.zip](https://github.com/plurimath/plurimath/files/13398871/equations.zip)

**Comment by opoudjis** [comment]
The problem here is that these likely have more to do with styling than content: the content seems to be fine.


The text spans are supposed to be text, but are ending up as the garbled:

````
<mtext>             <mrow>
              <mi>s</mi>
              <mi>y</mi>
              <mi>m</mi>
              <mi>m</mi>
              <mi>e</mi>
              <mi>t</mi>
              <mi>r</mi>
              <mi>i</mi>
              <mi>c</mi>
            </mrow>
          </mtext>
````

If I replace that with 

````
<mstyle mathvariant="italic"><mtext>symmetric</mtext></mstyle>
````

it renders correctly in HTML. And the sequence of mi's above is clearly bogus.

So need to find out why this is is not rendering as mtext properly.

**Comment by opoudjis** [comment]
... And THAT becomes @suleman-uzair ’s issue.

So, whatever mbox means in Latex, what we want is mtext in MathML, containing *just text*. Maybe it needs to be italicised, I have no idea, and I suspect that's a LaTeX idiosyncrasy. But `<mtext>` should contain text, it should not contain `<mrow>` and `<mi>`: the whole point of mtext is that its content is NOT mathematical elements.

**Comment by opoudjis** [comment]
The strange alignment in PDF is because jEuclid is being less generous than MathJax about what to do with table entries like "symmetric": MathJax is allowing them to spill over into the next column, jEuclid is taking them seriously.

It doesn't look to me like the LaTeX has anything like a notion of colspan, which means I don't think anything can be done here. But alerting @Intelligent2013 just in case.

**Comment by suleman-uzair** [comment]
@opoudjis, @manuelfuenmayor 

This issue is fixed in PR #200.
